tus-servers/file-server-https/readme.md

```diff
@@ -0,0 +1,24 @@
+### Create Cert for Local Dev
```

Comment:
Add a quick paragraph for the purpose of this server, steps to run locally, and some usage examples.